### DISCUSSION TITLE
Is there a way to synchronize brightness across all external monitors?

### DISCUSSION THREAD

**Discussion by MaryadaPurshottam** [MonitorControl/MonitorControl#1227] [Q&A] [answered]
I have 3 external monitors connected to my MacBook and was wondering if there was a way to synchronize brightness across all monitors so I don't individually have to change the brightness on all 3 one at a time.

**Comment by waydabber** [comment, marked as answer]
Sure, you can use a single slider for all 3 displays and also you can change the keyboard shortcuts to affect all 3 displays at once.

Take a look at these options:

Settings/App Menu/Multiple displays
Settings/Keyboard/Screen to control

Hope this helps!

**Comment by 1mike12** [comment]
![Screenshot 2024-07-28 at 09 53 20](https://github.com/user-attachments/assets/ceeb2d12-c9cc-43b9-9770-0a7b6556243c)
